WordPress/Tests/CodeAnalysis/EmptyStatementUnitTest.php

```diff
@@ -7,34 +7,27 @@
  * @license https://opensource.org/licenses/MIT MIT
  */
 
+namespace WordPress\Tests\CodeAnalysis;
```

Comment:
This file seems to be incorrectly renamed.

Reply:
That's just Git being weird, nothing to worry about.